src/main/java/uk/gov/justice/digital/client/kinesis/KinesisReader.java

```diff
@@ -24,33 +24,33 @@ public class KinesisReader {
     private final JavaDStream<byte[]> kinesisStream;
 
     @Inject
-    public KinesisReader(JobParameters jobParameters,
+    public KinesisReader(JobArguments jobArguments,
                          JobProperties jobProperties) {
         String jobName = jobProperties.getSparkJobName();
```

Comment:
no need to have this variable jobName and directly call the method in line 32 if this is used only in one place in the entire class